### ISSUE TITLE
Cannot submit a felt report.

### ISSUE THREAD

**Issue by radar57** [GeoNet/help#28]
Hi there. I am currently situated in Waikanae. In New Zealand. But for some reason the GeoNet iPad app seems to think I am in the US.  So I get this weird error every time I try to submit a Felt Report. It has only started doing this recently. The error reads as follows:

Domain=com.amazon was.AWSCognitoIdentity ErrorDomain="(null)" UserInfo={message=Access to Identity 'us-east-1:5de046a7-ffb6-4306-9f41-351d 198cf941' is forbidden., _type=NotAuthorizedException}

**Comment by junghao** [comment]
Would you please try remove GeoNet app and reinstall it? I'm guessing your app is stuck on previous login session.

**Comment by radar57** [comment]
Thank you for your response. I have uninstalled and reinstalled several times already but still get the same error. I have only ever submitted a felt report using my iPad from my home address in Waikanae so cannot think how it would confuse that with US.  I switched Location Services off and it asked me to turn it on so if it is using that and not a previous login, how can it not see where I am? 

Thanks for your help
